services/QuillLMS/app/models/cron.rb

```diff
@@ -89,6 +91,11 @@ def self.run_at_21_hour_mark
     AdminDiagnosticReports::PerformanceBenchmarkWorker.perform_async
   end
 
+  # 6/7PM depending on daylight savings
+  def self.run_at_22_hour_mark
```

Comment:
Relatedly, if this takes 10 minutes for 24 hours of data, we might consider running this more often during the day, e.g. if this takes a minute to run for an hour of data, maybe we should just run it every hour and issues would fix quicker (not sure if the runtime scales that way).

Reply:
I thought about this but was worried about adding unnecessary load to the system during peak hours.  However, it would probably be better from a user perspective since these lock issues could _potentially_  be fixed more quickly.  I will switch to hourly and monitor.

Reply:
I've updated the query to look at an hourly range, but I've shifted the window from 2.hours...1.hours rather than 1.hour...Time.current since I don't want to be calling currently completed ones twice.

https://github.com/empirical-org/Empirical-Core/pull/12607/files#diff-0624ca7f303f8e052d6a2c91703502f154e5dc027e59d4ccb6cbdacb19a0c5a2R15